explainer.md

```diff
@@ -72,6 +72,113 @@ There are still many open questions, but a few big ones:
 - Will existing protocols ([example](https://github.com/openid/OpenID4VP/issues/125)) adopt this API?
 - Will regulation ([example](https://digital-strategy.ec.europa.eu/en/library/european-digital-identity-architecture-and-reference-framework-outline)) adopt this API?
 
+# [Self-Review Questionnaire: Security and Privacy](https://w3ctag.github.io/security-questionnaire/)
+
+> 01.  What information does this feature expose,
+>      and for what purposes?
+
+The Digital Credential API exposes a one-time user-mediated communication channel from websites, to digital wallet applications, and back to websites with end-to-end encryption of the response. It is designed to be a better option than established lower-level communication channels like custom schemes, QR codes, and server-to-server network communication. How exactly this channel is used is up to the wallet applications and host operating system, but we are designing it to be suitable for conveying presentations of digital credentials such as claims in mobile driver’s licenses.
```

Comment:
Although it's good to talk about the "communication channel" through which the API works, this doesn't exactly answer what *information the feature exposes*. 

In the first sentence, I was expecting to read something like "high-value privacy sensitive digital credentials, such as digital driver's licenses... it does this through a one-time user-mediated communication channel from websites..." 

Generally, we should try other the answers specific to the questions being asked. We probably don't need to give justification for the decisions taken (i.e., why it's a better option than custom schemes, QR codes, etc), as we've covered those in other documents (which we should probably link to).

Reply:
Agreed with the first part and updated. 

I'm not sure I agree about only answering directly. We keep getting people presuming that we are seeking to enable exposure of this information (and arguing against the legitimacy of the use case), when in fact all we're doing is trying to replace existing exposure with safer mechanisms. So I'm just trying to head off the immediate criticisms I know we'll get. But perhaps I've overdone it?